### ISSUE TITLE
Allow for disabling of Interrupts

### ISSUE THREAD

**Issue by konfu** [adafruit/Adafruit-MCP23017-Arduino-Library#10]
There is only a function to enable interrupts per pin but none to disable interrupts.

Please offer functions to disable interrupt of a single pin or all pins altogether.

**Comment by PsuFan** [comment]
Yes, I agree. Ran into this too... Lost hours because a) MCP wasn't being reset when I was pushing esp8266 reset, b) I wasn't getting the correct interrupt pin and c) this function doesn't exist!

`void Adafruit_MCP23017::disableInterruptPin(uint8_t pin) {
  // disable the pin for interrupt
  updateRegisterBit(pin, LOW, MCP23017_GPINTENA, MCP23017_GPINTENB);
}
`

**Comment by ladyada** [comment]
hi please submit a PR with the fix!

**Comment by PsuFan** [comment]
Okay, I never submitted a pull before. Hope I did it right, #54

**Comment by PsuFan** [comment]
Sort of related: How do we feel about disabling all interrupts on begin()?

**Comment by ladyada** [comment]
thats fine

**Comment by caternuson** [comment]
Closing. Fixed with #54.
